### ISSUE TITLE
Coinex misses TPSL implicit api end points

### ISSUE THREAD

**Issue by BertkeOfUtrecht** [ccxt/ccxt#27909]
### Operating System

Windows

### Programming Language

Python

### CCXT Version

4.5.12

### Description

Below implicit apis for Coinex are missing and should be included 

v2_private_post_futures_cancel_position_stop_loss = v2PrivatePostFuturesCancelPositionStopLoss = Entry('futures/cancel-position-stop-loss', ['v2', 'private'], 'POST', {'cost': 20})
    v2_private_post_futures_cancel_position_take_profit = v2PrivatePostFuturesCancelPositionTakeProfit = Entry('futures/cancel-position-take-profit', ['v2', 'private'], 'POST', {'cost': 20})

**Comment by carlosmiei** [comment]
Hello @BertkeOfUtrecht,

These endpoints were added last week, can you please upgrade your ccxt version? if you do it you should be able to access them

https://github.com/ccxt/ccxt/pull/27833
